scripts/ELK/templates/metricbeat.yml

```diff
@@ -64,7 +64,7 @@ setup.kibana:
   # Scheme and port can be left out and will be set to the default (http and 5601)
   # In case you specify and additional path, the scheme is required: http://localhost:5601/path
   # IPv6 addresses should always be defined as: https://[2001:db8::1]:5601
-  host: "<KIBANA-MACHINE-PUBLIC-IP>:5601" # eg: "159.65.86.95:5601"
+  host: "167.99.47.69:5601" # eg: "159.65.86.95:5601"
```

Comment:
Aha, I assume there is no issue hardcoding the IP like this. Are we picking one for us?

Reply:
a crap. removed!